### PR TITLE
Feature/respond to an alert guide

### PR DESCRIPTION
**Pull request by timabrmsn**
@johnfpipkin provided this guide and I gave it an editing pass locally too.

### CODE REVIEW

File: source/respond-to-an-alert/index.html.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+curl -X POST "<RequestURL>/v1/alerts/query-alerts" \
+-H "accept: text/plain" \
+-H "Authorization: Bearer <AuthToken>" \
+-H "Content-Type: application/json" \
+-d '{ "tenantId": "<SampleTenant>", "groups": [ { "filters": [ { "term": "<FilterType>", "operator": "<OperatorValue>", "value": "<Criteria>" } ], "filterClause": "AND" } ], "groupClause": "OR", "pgSize": "20", "pgNum": "0", "srtKey": "CreatedAt", "srtDirection": "DESC" }'
+```
+
+In the preceding example:
+
+* Replace <RequestURL> with the [request URL](/intro-to-developer-portal/#request-urls) of your Code42 cloud instance.
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:98]: Inline HTML lines 98-100

Comment (unparalleled-js) [reply source/respond-to-an-alert/index.html.md:98]: Not really html, but you know I mean, just need to scape the `<`s

File: source/respond-to-an-alert/index.html.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+curl -X POST "<RequestURL>/v1/alerts/query-details" \
+-H "accept: text/plain" \
+-H "Authorization: Bearer <AuthToken>" \
+-H "Content-Type: application/json" \
+-d '{ "alertIds": [ "<SampleAlertID>" ]}'
+```
+
+In the preceding example:
+
+* Replace <RequestURL> with the [request URL](/intro-to-developer-portal/#request-urls) of your Code42 cloud instance.
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:141]: More Unescaped `<`'s

File: source/respond-to-an-alert/index.html.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    "description": "Sample description",
+    "findings": "Sample findings",
+    "subject":  <UserUID>,
+    "assignee": null
+}'
+```
+
+In the preceding example:
+
+* Replace <RequestURL> with the [request URL](/intro-to-developer-portal/#request-urls) of your Code42 cloud instance.
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:229]: Unescaped `<`s

File: source/respond-to-an-alert/index.html.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
+---
+title: Respond to an alert
+
+category: use-cases
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:4]: This is not showing up under use-cases for me

File: source/respond-to-an-alert/index.html.md [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+  
+Your company's entity in the Code42 cloud is known as a "tenant". To obtain your [tenant ID](/intro-to-developer-portal/#get-a-tenant-id), submit a request to the [customer API](/api/#tag/Customer).
+
+- **Code42 usernames for the people you want to add to alert rules** 
+  
+The names of employees you want to add to alert rules may come from a number of sources, such as an HR system or a directory service. While you can integrate with such systems using the Code42 API, remember that you need the Code42 username of these employees to be able to add them to alerts. The Code42 username is assigned when the user is [added to Code42](https://support.code42.com/Administrator/Cloud/Configuring/Add_users_from_the_Code42_console#Add_users_manually). You can obtain usernames via [CSV export](https://support.code42.com/Administrator/Cloud/Code42_console_reference/Users_reference#CSV_export), [user reports](https://support.code42.com/Administrator/Cloud/Code42_console_reference/User_Backup_report_reference#User_Backup_report), or the [user API](/intro-to-developer-portal/#get-useruids). 
+
+## Steps 
+
+### Search for significant alerts
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:84]: Missing newline under header

File: source/respond-to-an-alert/index.html.md [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+    "createdByUserUid": "806154242834341101",
+    "createdByUsername": "05c28ad3-7eca-4d99-8bb7-a5d0424206c8",
+    "lastModifiedByUserUid": "806154242834341101",
+    "lastModifiedByUsername": "05c28ad3-7eca-4d99-8bb7-a5d0424206c8"
+}
+```
+
+### Add file events to the case 
+
+To populate the new case with information about the alert, add file events from the alert to the case using the [/v1/cases/<CaseNumber>/fileevent/<EventID>](/api/#operation/addEventToCaseUsingPOST) API command. Repeat this action for all the file events from the alert.
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:257]: This link shows up looking like this: `/v1/cases//fileevent/ `

Comment (unparalleled-js) [reply source/respond-to-an-alert/index.html.md:257]: Probably just need to escape the <

File: source/respond-to-an-alert/index.html.md [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+
+```bash
+curl -X POST '<RequestURL>/v1/cases/<CaseNumber>/fileevent/<EventID>' \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer <AuthToken>'
+```
+
+In the preceding example:
+
+* Replace <RequestURL> with the [request URL](/intro-to-developer-portal/#request-urls) of your Code42 cloud instance.
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:267]: The final missing unescaped `<`s here

File: source/respond-to-an-alert/index.html.md [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+    "findings": "Sample findings",
+    "subject": null,
+    "subjectUsername": null,
+    "status": "OPEN",
+    "assignee": null,
+    "assigneeUsername": null,
+    "createdByUserUid": "806154242834341101",
+    "createdByUsername": "05c28ad3-7eca-4d99-8bb7-a5d0424206c8",
+    "lastModifiedByUserUid": "806154242834341101",
+    "lastModifiedByUsername": "05c28ad3-7eca-4d99-8bb7-a5d0424206c8"
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:251]: This does not look like a username

File: source/respond-to-an-alert/index.html.md [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+    "updatedAt": "2020-10-27T15:16:05.369203Z",
+    "description": "Sample description",
+    "findings": "Sample findings",
+    "subject": null,
+    "subjectUsername": null,
+    "status": "OPEN",
+    "assignee": null,
+    "assigneeUsername": null,
+    "createdByUserUid": "806154242834341101",
+    "createdByUsername": "05c28ad3-7eca-4d99-8bb7-a5d0424206c8",
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:249]: This also does not really look like a username

**Comment by unparalleled-js** [comment]
Not a blocker, but there is lot of trailing white spaces:

```
source/respond-to-an-alert/index.html.md:27: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:30: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:38: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:40: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:50: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:54: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:64: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:66: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:71: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:74: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:75: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:78: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:79: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:80: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:82: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:84: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:101: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:127: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:143: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:179: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:191: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:192: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:193: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:202: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:204: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:209: MD009 Trailing spaces
source/respond-to-an-alert/index.html.md:255: MD009 Trailing spaces
```

File: source/respond-to-an-alert/index.html.md [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+### Search for significant alerts 
+Alerts are an excellent way to get notifications about suspicious file activity. However, sometimes it can be difficult to separate the truly important ones from the rest.
+To find alerts that are worthy of further investigation, use the [/v1/alerts/query-alerts](/api/#operation/Alerts_QueryAlert) API command to search for alert notifications by alert filter criteria. By using filters you can uncover only the alerts that you want to follow up on.
+
+```bash
+curl -X POST "<RequestURL>/v1/alerts/query-alerts" \
+-H "accept: text/plain" \
+-H "Authorization: Bearer <AuthToken>" \
+-H "Content-Type: application/json" \
+-d '{ "tenantId": "<SampleTenant>", "groups": [ { "filters": [ { "term": "<FilterType>", "operator": "<OperatorValue>", "value": "<Criteria>" } ], "filterClause": "AND" } ], "groupClause": "OR", "pgSize": "20", "pgNum": "0", "srtKey": "CreatedAt", "srtDirection": "DESC" }'
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:93]: This request JSON is long

File: source/respond-to-an-alert/index.html.md [hunk 11]
```diff
@@ -0,0 +1,270 @@
...
+
+In the preceding example:
+
+* Replace <RequestURL> with the [request URL](/intro-to-developer-portal/#request-urls) of your Code42 cloud instance.
+* Replace <AuthToken> with the [authentication token](/intro-to-developer-portal/#authentication).
+* Replace <SampleAlertID> with the ID of an alert notification listed in the response in the "[Search for significant alerts](#search-for-significant-alerts)" section above. 
+
+A successful response returns full details about the alert notification, including event IDs (look for "eventid") and a list of files involved in the event (look for "files" followed by "name").
+
+```json
```
Comment (unparalleled-js) [inline source/respond-to-an-alert/index.html.md:147]: I notice you format with 4 spaces. I have been using 2 spaces.  Should we decide on one?

Comment (timabrmsn) [reply source/respond-to-an-alert/index.html.md:147]: I'm good with 2 spaces, I was just using the default for my note app which has a "prettify" button.

File: source/respond-to-an-alert/index.html.md (hunk 4 above)
Comment (timabrmsn) [reply source/respond-to-an-alert/index.html.md:4]: fixed

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**